orchagent/fabricportsorch.cpp

```diff
@@ -1220,5 +1241,24 @@ void FabricPortsOrch::doTask(swss::SelectableTimer &timer)
             updateFabricDebugCounters();
             updateFabricCapacity();
         }
+        if ((gMySwitchType == "fabric") && (!m_isSwitchStatsGenerated))
+        {
+            createSwitchDropCounters();
+            m_isSwitchStatsGenerated = true;
+        }
+    }
+}
+
+void FabricPortsOrch::createSwitchDropCounters(void)
```

Comment:
This function seems to be defined twice in 2 files, Can we put this under single file ? Also, we may call this twice for TM since m_isSwitchStatsGenerated flag is not updated on the TM side creation ?